changelogs/aws/v10.1.1.yaml

```diff
@@ -0,0 +1,75 @@
+- changelogs:
```

Comment:
I am not strictly against a file like this but having to put the correct version in the name is kind of difficult.

Additionally this file should be named `v10.1.2` to match the changes up top?

Reply:
I copied these changelogs from `10.1.1` so it should keep as it is I think. 
Why has the version name on the file is difficult? When it's wrong, you just need to fix it after that 😁 .

Reply:
The release notes also have the version in the filename so I think that should be OK. We could also check it exists with CI. A check that the app versions exist in the catalog would also be useful IMO.

But the filename being wrong in the example is confusing. So I think that should be fixed.

Reply:
@rossf7 Do you mean by changelogs/aws/v10.1.1.yaml? It's same changelogs from 10.1.1. Is there something wrong with versioning?

Reply:
In releases/aws.yaml you're adding `v10.1.2.yaml` but this file is named `v10.1.1.yaml`. I think it should also be named `v10.1.2.yaml`.

You can add a dummy changelog if you want. We can't actually merge this until the cluster-operator and cluster-service change are merged and we agreed with Ludacris we would wait until after the K8s 1.16 releases.

Reply:
I would add dummy 10.1.2.yaml for that. Thanks.